packages/flutter/lib/src/painting/continuous_rectangle_border.dart

```diff
@@ -107,15 +105,15 @@ class ContinuousRectangleBorder extends ShapeBorder {
     // We need to change the dimensions of the rect in the event that the
     // shape has a side width as the stroke is drawn centered on the border of
     // the shape instead of inside as with the rounded rect and stadium.
-    if (side.width > 0)
-      rect = rect.deflate(side.width / 2);
+    if (side.width > 0.0)
```

Comment:
stick to dart 2.1. No decimals unless it's useful.

Reply:
Wait Ian literally told me https://github.com/flutter/flutter/pull/26438#discussion_r258672366 to user 0.0 instead of 0.